index.js

```diff
@@ -0,0 +1,5 @@
+path = require('path')
+
+module.exports = (robot) => {
+  robot.loadFile(path.resolve(__dirname, 'src/reviewerQueue.js'))
```

Comment:
Path is necessary because this will become nested in node_modules. Also fixed the function call, was able to test my local hubot against the latest commit